index.html

```diff
@@ -9,10 +9,29 @@
 </head>
 <body>
     <header>
-        <nav class="menu">
-            <a href="#">John Doe</a>
-            <div><img src="./imgs/Union.svg" alt="hmb-button"></div>
-        </nav>
+            <div class="nav">
+                    <input type="checkbox" id="nav-check">
+                    <div class="nav-header">
+                      <div class="nav-title">
+                       <a href="#">GloireM</a> 
+                      </div>
+                    </div>
+                    <div class="nav-btn">
+                      <label for="nav-check">
+                        <span></span>
+                        <span></span>
+                        <span></span>
+                      </label>
+                    </div>
+                    
+                    <div class="nav-links">
+                      <a href="#">Portfolio</a>
+                      <a href="#">About</a>
+                      <a href="#">Contact</a>
+                      <a href="#"><img src="imgs/mail.svg" alt="Mail"></a>
+                      
+                    </div>
+                  </div>
         <div class="body-class">
```

Comment:
<img width="566" alt="Screenshot 2022-07-08 at 11 58 13" src="https://user-images.githubusercontent.com/51258047/177979625-a2519ee5-3d33-40fc-878c-594d1743b217.png">

Kindly place the My_footer div inside of a footer tag. It is important to use semantic tags to section your work for good/readability, for screen readers and SEO.